pkg/terminal/command.go

```diff
@@ -653,8 +666,45 @@ func writeGoroutineLong(w io.Writer, g *api.Goroutine, prefix string) {
 		prefix, formatLocation(g.GoStatementLoc))
 }
 
+func parseArgs(args string) ([]string, error) {
+	if args == "" {
+		return nil, nil
+	}
+	v, err := argv.Argv([]rune(args), argv.ParseEnv(os.Environ()),
```

Comment:
Do we gain a lot by introducing this new library? If so, we should refactor other argument parsing to take advantage of it.

Reply:
For this particular use case, we need a non-trivial argv parser to handle things like string quoting and escaping. This package is the simplest one I could find. I'm not sure if there could be other uses of this package.